torch/testing/_internal/common_methods_invocations.py

```diff
@@ -945,6 +945,7 @@ def method_tests():
         ('addr', (S, M), ((S,), (M,)), 'coef', (), (), (), ident, {'beta': 0.2, 'alpha': 0.6}),
         ('addr', (), ((S,), (M,)), 'broadcast_lhs_coef', (), (), (), ident, {'beta': 0.2, 'alpha': 0.6}),
         ('dot', (L,), ((L,),), '', (True,)),
+        ('vdot', (L,), ((L,),), '', (True,)),
```

Comment:
This should be (False,) at the end (to turn off JIT autodiff testing)

Reply:
Alternatively, `('vdot', (L,), ((L,),),` does the trick

Reply:
cc. @eellison `torch.vdot` is not supported by JIT autodiff right now.

Reply:
The question here is: Is it OK that autodiff doesn't support vdot? It's a new operator we added recently. Also, does the JIT still use the old autodiff pass?

Reply:
It should be fine if it's not supported by JIT autodiff - having an autodiff is an optimization which allows fusion to occur in training.  

Yea it still uses the old autodiff pass. The way to define a backwards is in the `symbolic_script.cpp` file. Currently, there is really only autodiff coverage for pointwise ops because those are the ops that we codegen fusion for.